mission-control/docs/guide/permissions/concepts/rbac.md

```diff
@@ -0,0 +1,24 @@
+---
+title: RBAC
+sidebar_position: 1
+---
+
+Role-Based Access Control (RBAC) is a security model that manages access to Mission Control resources based on the roles assigned to users.
+Mission Control provides a collection of built-in roles to accommodate common organizational needs and security requirements.
+When a user joins the system, they are assigned to one of these roles, establishing their base level of access
+
+| Role     | Description                                                                                                           |
+| -------- | --------------------------------------------------------------------------------------------------------------------- |
+| `admin`  | Provides full system access, including the ability to manage users, roles, and all resources.                         |
+| `editor` | Allows creation and modification of resources but does not permit managing users or system settings.                  |
+| `viewer` | Grants read-only access to resources.                                                                                 |
+| `guest`  | Offers minimal access with highly restricted viewing capabilities.<br></br> _([See Multi-Tenancy](multi-tenancy.md))_ |
+| `agent`  | Designed for agents, enabling them to fetch playbooks and push specific resources.                                    |
```

Comment:
@moshloop I have left out the `commander` and `responder` roles from here.

And, do we want to post the detail policy for each of these roles?